### ISSUE TITLE
Aller au travail autrement qu'en voiture

### ISSUE THREAD

**Issue by laloune** [datagir/nosgestesclimat#1457]
j'ai renseigné que je n'avais pas de voiture et que j'en utilise une uniquement pour les vacances, pourtant une des actions proposées est d'aller autrement en travail qu'en voiture... assez paradoxal

**Comment by laem** [comment]
Bonjour, merci pour le retor, en effet c'est bête. 
@Clemog peut-être qu'on peut exclure les non propriétaires ici ? https://github.com/datagir/nosgestesclimat/blob/master/data/transport/actions/transport.yaml#L182

Le seul pb que je vois c'est pour les covoitureurs. Il reste pertinent de leur connseiller d'aller autrement qu'en voiture au travail. Mais bon, c'est niche et ça diminue l'impact de l'action quand on covoiture surtout en tant qu'ajout de +1.

**Comment by Clemog** [comment]
Je ne pense pas qu'il faille exclure les non-propriétaires ... Peut-être qu'effectivement c'est juste pour un nombre restreint de personne. Ça se discute !

En revanche, le plus pertinent serait de récupérer l'info de l'aide à la saisie et rendre l'action applicable si un trajet correspond au "domicile-travail" mais en core faudrait-il que ce soit obligatoire de rentrer ses trajets .. @laloune je vois que tu précises "j'ai renseigné [...] que j'en utilise une uniquement pour les vacances", tu as renseigné tes trajets via le formulaire d'aide à la saisie ?

**Comment by laloune** [comment]
non, j'ai simplement cliqué sur "vacances", car je n'ai aucun idée du kilométrage annuel de mes trajets

**Comment by laloune** [comment]
concernant la partie "domicile-travail", pourquoi pas une option "je suis en majorité en télétravail", ce qui n'implique de facto pas de trajets domicile-travail
